RecoParticleFlow/PFClusterTools/interface/PFEnergyCalibration.h

```diff
@@ -93,15 +93,19 @@ class PFEnergyCalibration
   TF1* faBarrel;
   TF1* fbBarrel; 
   TF1* fcBarrel; 
-  TF1* faEtaBarrel; 
-  TF1* fbEtaBarrel; 
+  TF1* faEtaBarrelEH; 
+  TF1* fbEtaBarrelEH; 
+  TF1* faEtaBarrelH; 
+  TF1* fbEtaBarrelH; 
```

Comment:
please migrate these to smart pointers ```std::unique_ptr<TF1*>``` ; after doing that the ```delete ... ``` in the destructor will not be necessary.